constructorio-client/src/main/java/io/constructor/client/models/StatusCounts.java

```diff
@@ -0,0 +1,46 @@
+package io.constructor.client.models;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Constructor.io Status_Counts ... uses Gson/Reflection to load data in
+ */
+public class StatusCounts {
+
+    @SerializedName("QUEUED")
+    private int queued;
+
+    @SerializedName("DONE")
+    private int done;
+
+    @SerializedName("IN_PROGRESS")
+    private int inProgress;
+
+    @SerializedName("FAILED")
+    private int failed;
+
+    /**
+     * @return the QUEUED status count
+     */
+    public int getQueued() {
+        return queued;
+    }
+
+    /**
+     * @return the DONE status count
+     */
+    public int getDone() {
+        return done;
+    }
+
+    /**
+     * @return the IN_PROGRESS status count
+     */
+    public int getInProgress() { return inProgress; }
```

Comment:
Let's keep the function blocks consistent with functions above. Same with the fn below